snp_tpp/src/tpp_widget.cpp

```diff
@@ -84,9 +84,14 @@ void TPPWidget::callback(const snp_msgs::srv::GenerateToolPaths::Request::Shared
     mesh.header.frame_id = req->mesh_frame;
 
     // Plan the tool paths
-    std::vector<noether::ToolPaths> tool_paths = pipeline.plan(mesh);
+    std::vector<noether::ToolPaths> tool_paths_vector = pipeline.plan(mesh);
 
-    res->tool_paths = toMsg(tool_paths.at(0));
+    // Concatenate the tool paths from each sub-mesh into a single tool paths object
+    noether::ToolPaths tool_paths_concat;
+    for(const noether::ToolPaths& tool_paths : tool_paths_vector)
+      tool_paths_concat.insert(tool_paths_concat.end(), tool_paths.begin(), tool_paths.end());
+
+    res->tool_paths = toMsg(tool_paths_concat);
```

Comment:
@marrts what do you think about merging tool paths from different selection regions into one "super raster" for the motion planner? Essentially we would be using the transition planner to plan between regions. The alternative is to modify the planning service and server to plan for multiple raster regions, which seems a bit involved and potentially unnecessary (at least for now)

Reply:
Sounds like there are changes in the latest version of `tesseract` that we would like to incorporate into the motion planner configuration update, so let's merge this first